example1.ipynb

```diff
@@ -13,7 +13,7 @@
   },
```

Comment:
<p>I added. new header here to show the diff</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jamesdanielwhitford/notebook_diff_tutorial/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>